xml/containers-get.xml

```diff
@@ -0,0 +1,586 @@
+   <link xlink:href="https://github.com/GoogleContainerTools/container-diff#container-diff">container-diff</link>
+   a useful tool for analyzing and comparing container images.
+   <emphasis role="strong"><literal>container-diff</literal></emphasis> can
+   examine images along several different criteria, including:
+  </para>
+
+  <itemizedlist>
+   <listitem>
+    <para>
+     Docker Image History
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Image file system
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Apt packages
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     RPM packages
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     pip packages
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     npm packages
+    </para>
+   </listitem>
+  </itemizedlist>
+
+  <para>
+   These analyses can be performed on a single image, or a
+   <literal>diff</literal> can be performed on two images to compare.
+   <literal>container-diff</literal> supports Docker images located in both a
+   local Docker daemon and a remote registry. Tarballs with the correct
+   extension (.tar, .tar.gz, .tgz) can also be provided to the tool directly.
+  </para>
+
+  <para>
+   The <literal>container-diff</literal> package is part of the {sls} 15 SP2
+   <emphasis role="strong">Containers Module</emphasis> or it can be installed
+   separately. For instructions on how to install it, see the
+   <link xlink:href="https://github.com/GoogleContainerTools/container-diff#installation">container-diff
+   documentation</link>.
+  </para>
+ </sect1>
+ <sect1 xml:id="id-on-premise-registry">
+  <title>On-premise Registry</title>
+
+  <sect2 xml:id="id-portus">
+   <title>Portus</title>
+   <para>
+    <link xlink:href="http://port.us.org/">Portus</link> is an on-premise
+    application whose main purpose is to provide users with
+    <emphasis role="strong">authorization for their Docker
+    registries</emphasis>. The other main goal of Portus is to provide a
+    <emphasis role="strong">UI on top of your registry</emphasis>. It supports
+    LDAP user authentication and allows easy private registry management. For
+    further notable features of Portus, see:
+    <link xlink:href="http://port.us.org/features.html">http://port.us.org/features.html</link>
+   </para>
+   <para>
+    Portus can be deployed using a standard Docker container, but also inside a
+    Kubernetes cluster and even on Bare metal. For deployment options and
+    instructions on how to get started with exploring Portus inside a
+    development environment, see
+    <link xlink:href="http://port.us.org/docs/deploy.html">http://port.us.org/docs/deploy.html</link>.
+   </para>
+   <para>
+    For instructions on how to install Portus from the SUSE Container Registry
+    see the
+    <link xlink:href="https://documentation.suse.com/sles/12-SP4/html/SLES-all/cha-registry-installation.html#sec-docker-portus">{slsa}
+    Documentation</link>.
```

Comment:
If possible, make this an xref -- not sure here because the link points at 12 SP4 for some reason.
If no xref is possible, replace `{slsa}` -> `&slsa;`.